tests/parametric/test_span_events.py

```diff
@@ -9,7 +9,7 @@
 @rfc("https://docs.google.com/document/d/1cVod_VI7Yruq8U9dfMRFJd7npDu-uBpste2IB04GyaQ")
 @scenarios.parametric
 @features.span_events
-@missing_feature(reason="Agent does not advertise native span events serialization support yet")
+@missing_feature(True, reason="Agent does not advertise native span events serialization support yet")
```

Comment:
Is this change necessary? by not specifying a condition, it should be flagged.

Reply:
This change was require to use `assert condition is not None, _MANIFEST_ERROR_MESSAGE` with the missing_feature annotation. This is the only class annotation that sets condition=None.

Reply:
In a follow up PR I am planning to prevent folks from annotation classes with the bug/flaky/irrelevant/... decorators. These decorations should only be applied to methods. In all other scenarios the manifest file should be used.

Reply:
> These decorations should only be applied to methods. 

Unfortunately, no. Manifests can handle only a subset of what is possible with decorators. This subset covers 99% of use cases, but for the last bit, we need to keep this capacity. Exemples (not exhaustive) : 

```python
@bug(context.library == "python@1.2.3") # previous and next version are ok
@bug(context.library >= "python@1.2.3" and context.agent_version > "4.5.6") # integration is broken
class Test_NoLuck:
   ...
```